typings/index.d.ts

```diff
@@ -2226,10 +2231,10 @@ declare module 'discord.js' {
 
   type Base64String = string;
 
-  type BitFieldResolvable<T extends string> =
-    | RecursiveReadonlyArray<T | number | Readonly<BitField<T>>>
+  type BitFieldResolvable<T extends string, N extends number | bigint> =
+    | RecursiveReadonlyArray<T | number | bigint | Readonly<BitField<T>>>
     | T
-    | number
+    | N
     | Readonly<BitField<T>>;
```

Comment:
```suggestion
    | Readonly<BitField<T, N>>;
```
missing the second generic here